ee/docker-compose.ch.yml

```diff
@@ -87,6 +87,7 @@ services:
             - ..:/code
         restart: on-failure
         environment:
+            DEBUG: 1
```

Comment:
With the latest change @yakkomajuri did this will now load a ts-node-dev server right? That's not a good idea in docker.

Honestly though I think varying the script based on DEBUG is a bad idea in general. Perhaps for development we should just use a different command alltogether?

Reply:
I agree 100%

#7094